.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - 3.7
+  - 3.6
+
+install: pip install -U tox-travis
+
+script: tox -v
```

Comment:
This testing config isn't comprehensive.  Needs to run on linux, mac and windows. This will only run linux by default. Feel free to take inspiration from hangar's travis config if needed. 

Also may be worth looking into using github actions instead of travisCI? It's pretty nice once the initial setup is complete (and allows for many more concurrent jobs than travis (20 per repo instead of 5 per organization account on travis).

Reply:
This is in my TODO list for next minor release. Do you think I should work on this on 0.1.0 itself?

Reply:
I wouldn't publish 0.1 unless it works on all major platforms. Fixing platform specific issues is better handeld before the project gets too big. However, i suppose so long as it's documented as unix only and not pushed as a universal package to PyPi it is OK if this is more of a hassle then it's worth right now.